src/js/AppStoreMenu.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import Modal from 'react-modal'
+import { connect } from 'react-redux';
+import store from './store';
+import { addAppPendingSetAppProperties } from './actions'
+
+import AppHeader from './AppHeader';
+import StaticIcon from './Templates/Shared/StaticIcon';
+import ConfirmAlert from './ConfirmAlert';
+
+import BCController from './Controllers/BCController';
+import fileSystemController from './Controllers/FileSystemController';
```

Comment:
Please change name to `FileSystemController`

Reply:
fixed in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/b79ab73fcfd760a30437c25d53d1c6665ee29526